### PR TITLE
Refetch the minicheft subgraph info to update all data from ui

### PR DESCRIPTION
**Pull request by prodesert22**
- Refetch minicheft subgraph info when add/remove liquidity and add/remove farm
- Show earn detail when you have liquidity to able to remove

### CODE REVIEW

**Comment by HariSeldon23** [comment]
Task linked: [CU-32gz8xd After removing liq you have to refresh](https://app.clickup.com/t/32gz8xd)

File: src/pages/Beta/Pool/DetailModal/DetailView/index.tsx [hunk 1]
```diff
@@ -27,14 +27,17 @@ const DetailView = ({ stakingInfo, onDismiss, version }: PoolDetailProps) => {
...
+  const hasliquidity = isStaking || userPgl?.greaterThan('0') || liquidityInUSD > 0
   return (
     <>
       <MobileWrapper>
         <Header stakingInfo={stakingInfo} onClose={onDismiss} />
         <Box p={10}>
-          {isStaking && <EarnDetail stakingInfo={stakingInfo} version={version} />}
-          <Box mt={isStaking ? '10px' : '0px'}>
+          {hasliquidity && <EarnDetail stakingInfo={stakingInfo} version={version} />}
+          <Box mt={hasliquidity ? '10px' : '0px'}>
```
Comment (SarjuHansaliya) [inline src/pages/Beta/Pool/DetailModal/DetailView/index.tsx:40]: EarnDetail is only for users who are farming. By showing EarnDetail earn details to users who just have liquidity will confuse user.

File: src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx [hunk 2]
```diff
@@ -177,7 +180,7 @@ const AddLiquidity = ({ currencyA, currencyB, onComplete, onAddToFarm, type }: A
           ' ' +
           currencies[Field.CURRENCY_B]?.symbol
       })
-
+      queryClient.refetchQueries(['get-minichef-farms-v2', account])
```
Comment (SarjuHansaliya) [inline src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx:183]: lets create utility hook something like 
`const refetchMinichefSubgraph = useRefetchMinichefSubgraph()` 
which just wrap `queryClient.refetchQueries(['get-minichef-farms-v2', account])` so we don't have to repeat everytime.

File: src/pages/Beta/Pool/EarnWidget/Stake/index.tsx [hunk 3]
```diff
@@ -145,6 +148,7 @@ const Stake = ({ version, onComplete, type, stakingInfo, combinedApr }: StakePro
           addTransaction(response, {
             summary: t('earn.depositLiquidity')
           })
+          queryClient.refetchQueries(['get-minichef-farms-v2', account])
```
Comment (SarjuHansaliya) [inline src/pages/Beta/Pool/EarnWidget/Stake/index.tsx:151]: can you please try this once more ? This line didn't work for me when I did farm, It didn't updated info in left side.

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

**Comment by SarjuHansaliya** [comment]
We have moved pools components to components repo. I have taken these changes to components PR https://github.com/pangolindex/components/pull/79. So closing this PR

**Pull request closed without merging**